2017/CG-11-28.md

```diff
@@ -32,6 +32,32 @@ The meeting will be a Google Hangouts call.
        * Discussion of meeting time logistics
        * POLL: We should hold every third video call at an APAC friendly time.
        * POLL: We should time the meeting to avoid conflict with TC39.
+    1. The name of WebAssembly.Global
+       * See [WebAssembly/threads/issues/49](https://github.com/WebAssembly/threads/issues/49),
+         discussion stalled
+       * Poll-worthy?
+    1. The type of immutable globals exported from instances
+       * See [WebAssembly/threads/issues/73](https://github.com/WebAssembly/threads/issues/73),
+         discussion stalled
+       * Basically, either change existing API incompatibly or live with complex API
+       * Poll-worthy?
+    1. The type of the wake count argument and return value for atomic.wake
+       * Spec text changed this to i64 but poll at CG mildly in favor of older i32
+       * Would be good to nail this down now, code is landing in implementations
+       * Discussions bogged down on what TC39 may or may not want to do
+       * POLL: Revert to i32 with trapping in (the implausible) case of overflow
+    1. Conditional Segment Initalization instructions `init_memory`/`init_table`
+       * See [WebAssembly/threads/pull/79](https://github.com/WebAssembly/threads/pull/79)
+       * POLL: Should we limit these instructions to only being valid during instantiation?
+    1. Returning `WebAssembly.Global` object instead of Number for exported globals
+       * See [WebAssembly/threads/issues/73](https://github.com/WebAssembly/threads/issues/73)
+       * POLL: Do we want to have a uniform return value for exported immutable/mutable globals?
+       * POLL: Should we specify that immutable globals return `WebAssembly.Global`?
+    1. Require custom sections after all known sections
+       * See [WebAssembly/design/issues/1153](https://github.com/WebAssembly/design/issues/1153)
+       * We currently allow custom sections anywhere, but since most (all?) producers currently put
+         them at the end, we may have incompatibilities in implementations.
```

Comment:
What are the incompatibilities? I don't see any such mention in the bug.

Reply:
Luke was just speculating that it is likely undertested and likely to have problems. Should I omit?

Reply:
Yes.

Reply:
OK, we didn't get to this item anyway.